index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!--this is header section-->
+    <header class="header-container">
+        <h2 class="welcome">Welcome</h2>
+        <a class="menu-btn"><img src="/assets/images/Union.png" alt="menu button"></a>
+    </header>
+    <!--end of header section-->
+
+    <!--the main section-->
+    <main class="main-section-container">
+        <h2 class="main-title main-section-title">Hey there. I'm</h2>
+        <h2 class="main-title main-section-title">Maen</h2>    
+        <h2 class="main-title main-section-subtitle">I'm a software developer</h2>
+        
+        <p class="main-section-paragraph">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don&#39;t hestiate to contact me.
+        </p>
+        <section class="social-meadia-container">
+            <img class="social-meadia-item" src="assets/images/twitter.png" alt="twiter">
+            <img class="social-meadia-item" src="assets/images/Linkedin.png" alt="linkedin">
+            <img class="social-meadia-item" src="assets/images/medium.png" alt="medium">
+            <img class="social-meadia-item" src="assets/images/github.png" alt="github">
+            <img class="social-meadia-item" src="assets/images/angellist.png" alt="angellist">
```

Comment:
Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).